requirements-optional.txt

```diff
@@ -10,7 +10,11 @@ chardet>=2.2
 
 # lxml is supported with its own treebuilder ("lxml") and otherwise
 # uses the standard ElementTree support
-lxml ; platform_python_implementation == 'CPython'
+lxml ; platform_python_implementation == 'CPython' and sys_platform != 'win32'
+lxml==3.6.0 ; platform_python_implementation == 'CPython' and sys_platform == 'win32' and python_version <= '3.4'
+lxml==3.6.0 ; platform_python_implementation == 'CPython' and sys_platform == 'win_amd64' and python_version <= '3.4'
```

Comment:
It is not mandatory, but highly recommended to simplify life for Windows users.
My reasoning in the commit msg is:

> Require lxml 3.6.0 on Windows as it has wheels available for 2.6-3.4.

3.6.1 doesnt have wheels; if we dont pin it, pip will attempt to install 3.6.1 from source.

If you agree with this reasoning, ill add a comment explaining it.